test/test-applications/integrations/Samples.StartupHook/Samples.StartupHook.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0-rc.2.21480.5" />
```

Comment:
Microsoft.Extensions.Logging.Abstractions was released as 6.0.0.

Can we use it?